src/transaction.js

```diff
@@ -180,9 +180,9 @@ export class Transaction {
   }
 
   /**
-   * @private
+   * Get a buffer of the Transaction data that need to be covered by signatures
    */
-  _getSignData(): Buffer {
+  get signData(): Buffer {
```

Comment:
I think this should stay a function rather than act as a property. It's an expensive call